### DISCUSSION TITLE
https://notes.alexkehayias.com/using-chatgpt-with-emacs/

### DISCUSSION THREAD

**Discussion by giscus** [alexkehayias/note-comments#12] [Announcements]
# https://notes.alexkehayias.com/using-chatgpt-with-emacs/

There are a few packages and libraries that are being built to use ChatGPT along with Emacs.

https://notes.alexkehayias.com/using-chatgpt-with-emacs/

**Comment by pgpbpadilla** [comment]
Nice!
